testing/py-protobuf/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-protobuf
+_pkgname=protobuf
+pkgver=2.6.1
+pkgrel=0
+pkgdesc="Protocol Buffers are Google’s data interchange format."
+url="https://github.com/google/protobuf"
+arch="noarch"
+license="Copyright 2014, Google Inc. All rights reserved."
```

Comment:
This is semantically and even factually incorrect. [protobuf is licensed under BSD license](https://github.com/google/protobuf/blob/master/LICENSE), so there should be `license="BSD"`.